utils/build/docker/dotnet/app.csproj

```diff
@@ -17,7 +17,8 @@
 		<PackageReference Include="MySql.Data" Version="8.0.30" />
 		<PackageReference Include="Npgsql" Version="4.0.10" />
 		<PackageReference Include="System.Data.SqlClient" Version="4.6.1" />
-		<PackageReference Include="Microsoft.Data.Sqlite " />
+		<PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="7.0.15" />
+		<PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.1.8" />
```

Comment:
This repackages the same SQLLite driver but includes arm64 support?

Reply:
`Microsoft.Data.Sqlite` is built on top of `SQLitePCLRaw`, and the `SQLitePCLRaw.bundle_e_sqlite3` includes the arm64 binaries (by providing a consistent version of SQLite on all platforms). ([doc](https://learn.microsoft.com/en-us/dotnet/standard/data/sqlite/custom-versions?tabs=netcore-cli))